CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.1.0]
+### Changed
+- Bump `@metamask/eth-json-rpc-provider` from `^4.0.0` to `^4.1.0` ([#252](https://github.com/MetaMask/eth-block-tracker/pull/252))
+- Adapt to eip-1193 provider changes ([#252](https://github.com/MetaMask/eth-block-tracker/pull/252))
+  - Calling `sendAsync` function will be deprecated; so we used `request` function instead
```

Comment:
`sendAsync` is actually currently already marked as deprecated, can we rephrase to 
- The `request` method is now used instead of the deprecated `sendAsync` function.

Reply:
yes looks great for me but I am not sure if we use method or function in this case we used in the past method here https://github.com/MetaMask/core/blob/main/packages/eth-json-rpc-provider/CHANGELOG.md